person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :name, :age, :id
+
```

Comment:
- [ ] It's good that you have defined getter and setter for both name and age, but we are going to only call the `id` here so we only need `getter` for the `id` and not both.